languages/ruby/exercises/concept/arrays/.meta/design.md

```diff
@@ -0,0 +1,52 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of Arrays in Ruby.
+
+Of the many available Ruby collections, we chose to use the `array` collection type as the first collection type students will be taught for the following reasons:
+
+- Arrays are a common data type in many languages.
+- Arrays have a fixed length. No complexity in adding or removing elements.
+- Arrays have a simple shorthand syntax. No need to understand how initializers work to define an array.
+
+## Learning objectives
+
+- The existence of the `Array` object.
+- Defining an array.
+- Accessing elements in an array by index.
+- Iterating over elements in an array.
+- Basic array methods (like size or sum).
+
+## Out of scope
+
+- Multi-dimensional/jagged arrays.
+- Memory and performance characteristics of arrays.
+- Iterators.
+
+## Concepts
+
+This Concepts Exercise's Concepts are:
+
+- `arrays`: know of the existence of the `Array` object; know how to define an array; know how to access elements in an array by index; know how to iterate over elements in an array; know of some basic functions.
+- `each loops`: know how to iterate over a collection.
```

Comment:
The `config.json` has updated the concepts, which should also be reflected here.